### PR TITLE
Dev/221 [ExamQuestion] Change exam questions by mouse scrolling

### PR DESCRIPTION
**Pull request by Wally5077**
resolve: #221 

https://user-images.githubusercontent.com/26381900/136546465-f76af49b-9ffa-408a-aae7-408892179529.mov

### CODE REVIEW

File: src/app/commons/ItemListPage/ItemListPage.js [hunk 1]
```diff
@@ -27,12 +27,19 @@ const FilterSearchBar = ({filterItems, Button}) => {
  *  receives an index of the data and then return the content (html) of that within the <td></td>
  *  (3) key: a function that determines the key of the item given a student
  * @param tableDataStyle the custom style of the <td> elements
+ * @param showFilterSearchBar the decision (boolean) of showing default <FilterSearchBar> component
+ * @param draggable the decision (boolean) of <tr> draggable elements
+ * @param ondrag the operation (obj) of <tr> elements dragging
+ * @param ondrop the operation (obj) of <tr> elements dropping
```
Comment (Johnny850807) [inline src/app/commons/ItemListPage/ItemListPage.js:33]: Please help correct this to the JsDoc writing format.

You can reference `ConfirmationModal`'s JsDoc

**Review by Chaoyuuu — APPROVED** [review]
LGTM

File: src/app/commons/ItemListPage/ItemListPage.js [hunk 2]
```diff
@@ -17,29 +17,37 @@ const FilterSearchBar = ({filterItems, Button}) => {
...
 };
 
-
 /**
- * @param title the title (str) of the page
- * @param filterItems the item's name (str) that is included in the filter drop-down list
- * @param onCreateButtonClick the event listener triggered when the green 'Create' button get clicked
- * @param tableHeaders a list of header names (str) in the table
- * @param tableRowGenerator 3 fields (1) list: 'the data list' (2) data: a data generation function that
+ * @callback callback
```
Comment (Johnny850807) [inline src/app/commons/ItemListPage/ItemListPage.js:21]: This should be incorrect, you should declare the callbacks in another comment section and you also need to declare its parameters.  see: https://jsdoc.app/tags-callback.html

Comment (Johnny850807) [reply src/app/commons/ItemListPage/ItemListPage.js:26]: @Wally5077 unresolve

**Pull request merged**